exist-core/src/main/resources/org/exist/xquery/lib/xqsuite/xqsuite.xql

```diff
@@ -511,12 +512,12 @@ declare %private function test:map-named-arguments($funcArgs as element(argument
             $mappedArgs
 };
 
-declare %private function test:map-named-argument($testArgs as element(value)*, $funcArg as element(argument)) {
+declare %private function test:map-named-argument($test-args as element(value)*, $func-arg as element(argument)) {
     let $data :=
         try {
-            test:cast($testArgs, $funcArg)
+            test:cast($test-args, $func-arg)
         } catch * {
-            error($test:TYPE_ERROR, "Failed to cast annotation arguments to required target type " || $funcArg/@type)
+            fn:error($test:TYPE_ERROR, "Failed to cast annotation arguments to required target type " || $func-arg/@type)
```

Comment:
Why? What do we gain for doing that?

Reply:
Clarity. It's unnnecessary clutter.

Reply:
I think that is subjective, personally I have been writing all XQuery code for sometime now using the `fn` prefix as we have found here that it increases clarity rather than reducing it.

Reply:
I will put it on the agenda for the next community meeting then.

Reply:
Yeah, we need a definitive way forward. The entire codebase does not use the prefix so it is necessary to decide whether it is used _everywhere_ in the future or _never_.

Reply:
> The entire codebase does not use the prefix

If you grep for `fn:` you will find it in plenty of XQuery files within eXist-db!

Reply:
The status quo is troubling, I agree.
This PR adds prefixes to existing lines of codes that are otherwise unchanged and the use of prefixes is consistent within the module itself.